docs/source/examples/auto-failover.rst

```diff
@@ -76,41 +70,112 @@ AWS, where it succeeded after two regions:
-  W 02-23 16:42:26 cloud_vm_ray_backend.py:479]   create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient p3.16xlarge capacity in the Availability Zone you requested (us-east-2b). Our system will be working on provisioning additional capacity. You can currently get p3.16xlarge capacity by not specifying an Availability Zone in your request or choosing us-east-2a., retrying.
-  W 02-23 16:42:26 cloud_vm_ray_backend.py:479]   create_instances: Attempt failed with An error occurred (Unsupported) when calling the RunInstances operation: Your requested instance type (p3.16xlarge) is not supported in your requested Availability Zone (us-east-2c). Please retry your request by not specifying an Availability Zone or choosing us-east-2a, us-east-2b., retrying.
-  W 02-23 16:42:26 cloud_vm_ray_backend.py:479]   create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient p3.16xlarge capacity in the Availability Zone you requested (us-east-2a). Our system will be working on provisioning additional capacity. You can currently get p3.16xlarge capacity by not specifying an Availability Zone in your request or choosing us-east-2b., retrying.
-  W 02-23 16:42:26 cloud_vm_ray_backend.py:479]   botocore.exceptions.ClientError: An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient p3.16xlarge capacity in the Availability Zone you requested (us-east-2b). Our system will be working on provisioning additional capacity. You can currently get p3.16xlarge capacity by not specifying an Availability Zone in your request or choosing us-east-2a.
+  ...
   I 02-23 16:42:26 cloud_vm_ray_backend.py:668]
   I 02-23 16:42:26 cloud_vm_ray_backend.py:668] Launching on AWS us-west-2 (us-west-2a,us-west-2b,us-west-2c,us-west-2d)
   I 02-23 16:47:04 cloud_vm_ray_backend.py:740] Successfully provisioned or found existing VM. Setup completed.
+
+
+Multiple Candidate GPUs
+-------------------------
+
+If a task can be run on different GPUs, the user can specific multiple candidate GPUs,
+and SkyPilot will automatically find the best available GPU.
+
+To allow SkyPilot to choose any of the candidate GPUs, specify a set of candidate GPUs in the task yaml:
+
+.. code-block:: yaml
+
+  resources:
+    accelerators: {A10:1, L4:1, A10g:1}
+
+In the above example, SkyPilot will try to provision the any cheapest available GPU within the set of
+A10, L4, and A10g GPUs, using :code:`sky launch task.yaml`.
+
+.. code-block:: console
+
+  $ sky launch task.yaml
+  ...
+  -----------------------------------------------------------------------------------------------------
```

Comment:
Should we have the same indent for this table row separator too? Also, might be fine to remove the logging line (i.e., `SKYPILOT_MINIMIZE_LOGGING=1`)
```suggestion
I 11-19 08:00:03 optimizer.py:910]  -----------------------------------------------------------------------------------------------------
```

Same below

Reply:
Thanks for catching this! I tried `SKYPILOT_MINIMIZE_LOGGING` here, but the lines are a bit hard to distinguish from each other, as the color cannot be shown in the block. I make the output align with the prefix instead. : )